test/ffmpeg-vaapi/transcode/avc.py

```diff
@@ -0,0 +1,69 @@
+###
+### Copyright (C) 2018-2019 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from ....lib import *
+from ..util import *
+from .transcoder import TranscoderTest
+
+spec = load_test_spec("avc", "transcode")
+@platform_tags(set(AVC_DECODE_PLATFORMS)&set(AVC_ENCODE_PLATFORMS))
+class AVC(TranscoderTest):
```

Comment:
Maybe ```class to_avc``` would be more descriptive (and similar for other classes).  Then, printed test names would look something like:

```
test/ffmpeg-vaapi/transcode/avc.py:to_avc.test_hwhw(...)
test/ffmpeg-vaapi/transcode/avc.py:to_avc.test_hwsw(...)
test/ffmepg-vaapi/transcode/avc.py:to_hevc.test_hwhw(...)

etc..
```